light-zk.js/src/utxo.ts

```diff
@@ -132,16 +132,16 @@ export class Utxo {
     verifierProgramLookupTable: string[];
   }) {
     // check that blinding is 31 bytes
-    try {
-      blinding.toArray("be", 31);
-    } catch (_) {
-      throw new UtxoError(
-        UtxoErrorCode.BLINDING_EXCEEDS_SIZE,
-        "constructor",
+    // try {
+    //   blinding.toArray("be", 31);
+    // } catch (_) {
+    //   throw new UtxoError(
+    //     UtxoErrorCode.BLINDING_EXCEEDS_SIZE,
+    //     "constructor",
 
-        `Bliding ${blinding}, exceeds size of 31 bytes/248 bit.`,
-      );
-    }
+    //     `Bliding ${blinding}, exceeds size of 31 bytes/248 bit.`,
+    //   );
+    // }
```

Comment:
can this be removed?

Reply:
Good point I replaced the check with a better one